subsys/llext/llext.c

```diff
@@ -221,3 +221,37 @@ int llext_call_fn(struct llext *ext, const char *sym_name)
 
 	return 0;
 }
+
+void llext_bootstrap(struct llext *ext, llext_entry_fn_t entry_fn, void *user_data)
+{
+	int ret = llext_get_init_array(ext, NULL, 0);
+
+	if (ret < 0) {
+		LOG_ERR("Failed to get init array count");
```

Comment:
Definitely not possible to print the name, this can be run in user context so the `struct llext` content isn't available. I can print the pointer, but would not be the same :slightly_smiling_face:

Reply:
Fair enough.